src/components/Book.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Book({ title, author }) {
+  Book.propTypes = {
+    title: PropTypes.string.isRequired,
+    author: PropTypes.string.isRequired,
+  };
+
+  return (
+    <>
+      <ul>
+        <li>
+          {title}
+          :
+          {author}
+        </li>
```

Comment:
- [ ] **Required:** You are required to render a list of books.

![image](https://user-images.githubusercontent.com/27709832/171911395-a1edeb2e-55bb-4930-8aff-db608c43a836.png)

However, this is what renders to your page. Add some books.
![image](https://user-images.githubusercontent.com/27709832/171911143-5a0736ec-6435-4751-a9b4-7ef4fea651c8.png)

Reply:
@brytebee Thank you for your suggestions & for encouraging me to try again.
 Is it ok if I used a hook to store a list of random books in an object and render that object?

Reply:
@NatiG25 You could as long as books display on your page on every render.

Reply:
@brytebee I'm sorry for the late reply but I think you got the wrong requirements. Forgive me for asking, but are you sure that these are the requirements for Module 3, Week 2, and Day 4 project?

I've done the implementations if you can please check it, thank you.

Reply:
@brytebee Hello again, I've checked with the requirements & it says to only render one book. So, I did the implementation but only rendered one book. 

Forgive me for asking, but can you please make sure if those are the actual requirements for Module 3, Week 2, and Day 4 project?